include/igraph_strvector.h

```diff
@@ -56,7 +57,7 @@ typedef struct s_igraph_strvector {
  *
  * Time complexity: O(1).
  */
-#define STR(sv,i) ((const char *)((sv).stor_begin[(i)]))
+#define STR(sv,i) igraph_strvector_get(&sv, i)
```

Comment:
Is this a breaking change? This can surely not be used as an lvalue any more, but maybe the earlier version was not suitable for an lvalue either due to casting?

Reply:
You are right, this is not a breaking change.